README.md

```diff
@@ -34,7 +34,7 @@ Feel free to create an issue/PR if you want to see anything else implemented.
 
 ```lua
 use {
-    "Exafunction/codeium.nvim",
+    "Exafunction/windsurf.nvim",
```

Comment:
This change suggests that the GitHub repo name has changed, but it did not. This snippet for installation therefore no longer works.